README.md

```diff
@@ -1,24 +1,216 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+ Budget App
 
-* Ruby version
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 budget App ](#-budget-app-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Budget App <a name="about-project"></a>
 
-* Database creation
 
-* Database initialization
+This project is about building a mobile web application to manage a person budget. He will provide list of transactions associated with a category, so that he can see how much money he spent against each transacction.To keep data private, user go through authentication process.
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
+<div align="center">
 
-* Deployment instructions
+  <img src="./app/assets/images/diagram.png" alt="Logo" />
+  <br/>
 
-* ...
+</div>
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server Side</summary>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/en/">Ruby on Rails</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Splash Screen**
+- **Sign up and log in pages**
+- **Home page (categories page)**
+- **Transactions page**
+- **Add a new category page**
+- **Add a new transaction page**
+- **Use devise for user authentication**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- <a href="https://muneeb.onrender.com/">Live Demo</a>
```

Comment:
- [ ] `Required`
- [ ] Amazing Job so far! 👏 
- [ ] The Project is deployed okay but, I keep on getting an error after I sign up 😞 
- [ ] Please see to it the deployment is working okay 🥂 
- [ ] Added a screenshot of the error below
![image](https://github.com/muneeb3123/Budget-App/assets/66230306/0279b128-d5c7-4da0-96ec-cb1818f3e64c)

Reply:
In my case its working properly   i attached video in following

https://github.com/muneeb3123/Budget-App/assets/118625824/44ece68e-5f5d-4e00-bf40-4420857ae0da